src/Writers/LatexWriter.jl

```diff
@@ -0,0 +1,24 @@
+"""
+Provides the [`render`](@ref) methods to write the documentation as Latex files
+(`MIME"text/latex"`).
+"""
+module LatexWriter
```

Comment:
Just noticed, this should be `LaTeXWriter` to follow official spelling for the formats, i.e. `HTMLWriter` not `HtmlWriter`. Other than that, looks ready to merge to me.